test/source/tests/page-recipe/setup-page-recipe.ts

```diff
@@ -260,7 +260,9 @@ export class SetupPageRecipe extends PageRecipe {
         expect(await settingsPage.isChecked('@input-step2ekm-save-passphrase')).to.equal(enterPp.checks.isSavePassphraseChecked);
       }
       await settingsPage.waitAndClick('@input-step2ekm-continue');
-      await settingsPage.waitAndRespondToModal('confirm-checkbox', 'confirm', 'Please write down your pass phrase');
+      if (enterPp.checks?.postitNoteReminder !== false) {
```

Comment:
I wouldn't classify `postitNoteReminder` as an "optional check"  because optional checks are supposed to produce no errors when `undefined` is passed. However, when I run any of the tests you modified with `postitNoteReminder: undefined`, they fail. Hence this parameter should be kept separately from optional checks and better be named `expectPostitNoteReminder`. As it's logically connected to `enterPp`, it can be put inside `enterPp`.

Reply:
Done, thanks for the suggestion! 👍

Reply:
Thank you... but... will we ever need to set `expectPostitNoteReminder: true` for `autoKeygen` recipe? This recipe is for ekm tests only, as we have `createKey` recipe for key generation and `manualEnter` for import etc.
I guess this `if` block can be removed completely. Sorry I haven't noticed it yesterday.